content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -153,7 +153,11 @@ If you do not already have a hypervisor installed, install one of these now:
 • [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
 
 {{< note >}}
+
+**Hyper-V**
```

Comment:
Maybe remove this line?

Reply:
Thanks, and sorry for not cleaning this up earlier. Removed with latest push.